src/openstack_manager.py

```diff
@@ -113,3 +85,196 @@ def list_projects(cloud_config: dict) -> list[Project]:
         ) from exc
 
     return projects
+
+
+class ImageBuildError(Exception):
+    """Exception representing an error during image build process."""
```

Comment:
Sometimes errors are in `errors`, and sometimes errors are in their respective modules. I think we should agree on an approach here.

Reply:
You're right, thanks for catching this!